_translations/en.yml

```diff
@@ -31,12 +31,8 @@ en:
     title: "Bitcoin for Businesses - Bitcoin"
     pagetitle: "Bitcoin for Businesses"
     summary: "Bitcoin is a very secure and inexpensive way to handle payments."
-    lowfee: "The lowest fees out there"
-    lowfeetext: "Bitcoin's high cryptographic security allows it to process transactions in a very efficient and inexpensive way. You can make and receive payments using the Bitcoin network with almost no fees. In most cases, fees are not strictly required but they are recommended for faster confirmation of your transaction."
```

Comment:
>The lowest fees out there
change to
>Negational, mostly low fees.

And the last sentence should stay there, with this adjusted

>......In most cases, fees are not strictly required but they are recommended for faster confirmation of your transaction..

change to

> .......In almost all cases, fees are required for a reasonable duration of confirmation of your transaction....